tests/utils/mock_server.py

```diff
@@ -297,7 +297,6 @@ def graphql():
         if body["variables"].get("files"):
             requested_file = body["variables"]["files"][0]
             ctx["requested_file"] = requested_file
-            app.logger.info("graphql requested file: %s", requested_file)
```

Comment:
This introduced flakeyness to a parallel runs test.

Reply:
Wait, how was this related to "flakeyness"?

Reply:
I honestly don't know. But with that line I was seeing about a 10% error rate on `test_parallel_runs`.